packages/grafana-ui/src/components/FileUpload/FileUpload.test.tsx

```diff
@@ -29,4 +31,19 @@ describe('FileUpload', () => {
     });
     expect(wrapper.find({ 'aria-label': 'File name' }).text()).toEqual('longFileName.png');
```

Comment:
🤔 should we remove/update the enzyme test above in this PR?

Reply:
@axelavargas I rewrote the tests to use react testing library instead of enzyme